### PR TITLE
Api favorite book

### PR DESCRIPTION
**Pull request by mvn-nale-dn**
## Description
Api favorite book
## Reference Requirements
- Task ID: #203

## Reference Isuees
#{issue_id}

## Migrations
YES | NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Comment by at-haytran** [comment]
LGTM

File: app/Http/Controllers/API/FavoriteController.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+     * Api update status
+     *
+     * @param \App\Models\Favorite       $favorite favorite of this favorite
+     * @param \App\Http\Requests\Request $request  request
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function update(Favorite $favorite, Request $request)
+    {
+        $favorite = Favorite::findOrFail($favorite->id);
```
Comment (at-vietpham) [inline app/Http/Controllers/API/FavoriteController.php]: Đã dùng blinding model rồi không cần phải `findOrFail` lại nữa.

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by at-haytran** [comment]
LGTM

**Comment by at-vietpham** [comment]
LGTM 🚶

**Pull request merged**